python/test/Ice/asyncio/TestI.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) ZeroC, Inc. All rights reserved.
+#
+
+import asyncio
+import Ice
+import Test
+
+
+class TestIntfI(Test.TestIntf):
+
+    async def op(self, current):
+        return 5
+
+    async def throwUserException1(self, current):
+        raise Test.TestException()
+
```

Comment:
Please add some comments to the code below. With all these exception operations, it's not clear what is expected and what is not. Is `await asyncio.sleep(0.01)` in a dispatch coroutine expected to succeed? Is the sleep dispatch different from the other asyncio.sleep?

Reply:
I've added some comments.